Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.2
+MAINTAINER dengnan@google.com vmarmol@google.com vishnuk@google.com jimmidyson@gmail.com
+
+WORKDIR /gopath/src/github.com/google/cadvisor
+COPY    . /gopath/src/github.com/google/cadvisor
+
+RUN apk add --update --repository=http://dl-4.alpinelinux.org/alpine/edge/community/ -t build-deps go sqlite-dev btrfs-progs-dev bash linux-headers make git gcc g++ && \
```

Comment:
Interesting.  I hadn't considered some of the implications of the way docker handles layers.  Does this imply that cleaning up data from previous layers does not reduce the aggregate image size.  How important is image size for cadvisor?  Is there a threshold we want to keep it under?  FWIW, the git src looks like it consumes ~30MB, while the image "virtual size" is ~65MB, so copying the source and building cadvisor within the Dockerfile will increase the image size by 30MB.  (not sure if this is considered a lot or trivial)

```
$ docker exec -ti e58b09c05e64 du -h -d 1 /gopath/
29.7M   /gopath/src
9.8M    /gopath/bin
764.0K  /gopath/pkg
40.3M   /gopath/
```

```
$ docker images
REPOSITORY                                        TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
pwittrock/cadvisor                                latest              763321a5339a        About an hour ago   62.56 MB
```